llvm/include/llvm/CodeGen/MachineFunction.h

```diff
@@ -239,6 +239,12 @@ class MachineFunctionProperties {
     return *this;
   }
 
+  // Set all the properties.
+  MachineFunctionProperties &set() {
+    Properties.set();
+    return *this;
+  }
```

Comment:
This seems not useful and a hazard. No pass is going to change all of the properties

Reply:
Fixed.